utils/jira.js

```diff
@@ -76,11 +68,14 @@ function getVersions(project) {
  */
 function getProjectIdByKey(key) {
   return jira
-    .getProject(key)
+    .getProject(key.toUpperCase())
     .then((response) => {
       return response.id
     })
-    .catch((err) => console.log(`${err}`))
+    .catch((err) => {
+      console.log(`${err}`)
```

Comment:
Is the intention to swallow all these errors and log them? Seems like you would want to exit with an error if the project can't be found?

Reply:
Short answer is yes, so that it can sync what it can but not fail completely.